gnocchi/statsd.py

```diff
@@ -110,6 +113,7 @@ def flush(self):
                         archive_policy_name=ap_name,
                         name=metric_name,
                         resource_id=self.conf.statsd.resource_id)
```

Comment:
can do this in another patch but we should try to catch NamedMetricAlreadyExists if someone creates metric via api